tools/firmware.toit

```diff
@@ -368,23 +368,23 @@ extract parsed/cli.Parsed -> none:
   write_file output_path: it.write content
 
 extract_binary envelope/Envelope -> ByteArray:
-  firmware_bin/ByteArray? := null
-  properties/Map := {:}
   containers ::= []
-
   entries := envelope.entries
+
+  properties_entry := entries.get AR_ENTRY_PROPERTIES
+  properties := properties_entry ? (json.decode properties_entry) : null
   system := entries.get AR_ENTRY_SYSTEM_SNAPSHOT
-  if system: containers.add (ContainerEntry "system" system --assets=null)
+  if system:
+    wifi := properties.get "wifi"
```

Comment:
Should there be a warning if there are other properties?

Reply:
I think I will drop the support for properties as soon as we make it easier to work with assets. Let me think about this.